Src/AutoMoq/StubPropertiesCommand.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Moq;
+using Ploeh.AutoFixture.Kernel;
+
+namespace Ploeh.AutoFixture.AutoMoq
+{
+    /// <summary>
+    /// Stubs a mocked object's properties, giving them "property behavior".
+    /// Setting a property's value will cause it to be saved and later returned when the property is accessed.
+    /// </summary>
+    public class StubPropertiesCommand : ISpecimenCommand
+    {
+        /// <summary>
+        /// Stubs a mocked object's properties, giving them "property behavior".
+        /// Setting a property's value will cause it to be saved and later returned when the property is accessed.
+        /// </summary>
+        /// <param name="specimen">The mock to setup.</param>
+        /// <param name="context">The context of the mock.</param>
+        public void Execute(object specimen, ISpecimenContext context)
+        {
+            if (specimen == null) throw new ArgumentNullException("specimen");
```

Comment:
Perhaps we should just keep it as is for consistency... Apparently, it's not an issue.